android/capacitor/src/main/java/com/getcapacitor/FileUtils.java

```diff
@@ -134,6 +137,26 @@ else if ("file".equalsIgnoreCase(uri.getScheme())) {
         return null;
     }
 
+    /**
+     * Read a plaintext file.
+     *
+     * @param context Used to get access to the asset manager to open the file.
+     * @param fileName The path of the file to read.
+     * @return The contents of the file path.
+     * @throws IOException Thrown if any issues reading the provided file path.
+     */
+    static String readFile(Context context, String fileName) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(context.getAssets().open(fileName)));
```

Comment:
`reader.close()` is never called. new code should use try-with-resources: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html